gateway-ha/src/main/java/io/trino/gateway/ha/persistence/dao/QueryHistoryDao.java

```diff
@@ -62,7 +62,7 @@ SELECT count(1) FROM query_history
     Long count(@Define("condition") String condition);
 
     @SqlQuery("""
-            SELECT CONCAT(FLOOR(created / 1000 / 60)) AS minute,
+            SELECT FLOOR(created / 1000 / 60) AS minute,
```

Comment:
Can you test with PostgreSQL and MySQL and share the screenshot of Web UI?

Reply:
Can't see screenshot

Reply:
Thanks for testing, but the screenshot doesn't cover the actual situation. We should run a query before accessing the UI. 

Note that current change causes NumberFormatException if the table isn't an empty.